molo/core/templates/admin/date_range_filter.html

```diff
@@ -3,15 +3,8 @@
   {% include "wagtailadmin/shared/datetimepicker_translations.html" %}
   <script type="text/javascript">
   $(function() {
-    $('#id_drf__submission_date__gte').datetimepicker({
-      timepicker: false,
-      format: 'Y-m-d',
-      i18n: {
-          lang: window.dateTimePickerTranslations
-      },
-      lang: 'lang'
-    });
-    $('#id_drf__submission_date__lte').datetimepicker({
+    console.log("hey");
```

Comment:
@projectmushroom can we remove this in a future PR please? 😆

Reply:
haha, oh snap thought I cleaned it!